dev-libs/rocm-opencl-runtime/files/rocclr-5.3.3-gcc13.patch

```diff
@@ -0,0 +1,25 @@
+From ca70cd7c3d63b1036e604116f3b269ca123e7904 Mon Sep 17 00:00:00 2001
```

Comment:
Please always include a link to the upstream commit(s) and any relevant references (including Gentoo bugs) at the top of patches.

Thank you! I'll merge anyway in this case but going forward, it'd be a big help.

Reply:
I'll add the link to the upstream commit

Reply:
Oh I see you already closed it. I'll put the link and my name in maintainer list in another PR